### PR TITLE
Import python implementations from python, not rust wrapper

### PR DESCRIPTION
**Pull request by alexgleith**
## Closes

- Closes https://github.com/stac-utils/stac-rs/issues/462

## Description

Fixes the main import for the __init__.py file by importing the Python implementations rather than the naked rust wrappers of them.

## Checklist

Delete any checklist items that do not apply (e.g. if your change is minor, it may not require documentation updates).

- [ ] Unit tests
- [ ] Documentation, including doctests
- [ ] Git history is linear
- [ ] Commit messages are descriptive
- [ ] (optional) Git commit messages follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Code is formatted (`cargo fmt`)
- [ ] `cargo test`
- [ ] Changes are added to the CHANGELOG

### CODE REVIEW

File: python/python/stacrs/__init__.py [hunk 1]
```diff
@@ -1,16 +1,19 @@
-from . import stacrs
-from .stacrs import (
+import pystac
```
Comment (gadomski) [inline python/python/stacrs/__init__.py:1]: It was an intentional decision to _not_ import **pystac** at the top level — I'd like this library to be zero-dependency by default.

File: python/python/stacrs/pystac.py
Comment (gadomski) [inline python/python/stacrs/pystac.py:1]: We'd want a test to confirm that this works.

**Review by gadomski — CHANGES_REQUESTED** [review]
Thanks for the contrib @alexgleith! Looks like there's a couple issues w/ imports and tests — I might look to tackle https://github.com/stac-utils/stac-rs/issues/462 in a separate PR, I'll be sure to cc you in.

File: python/python/stacrs/__init__.py (hunk 1 above)
Comment (alexgleith) [reply python/python/stacrs/__init__.py:1]: Oh, sorry, that was meant to be the internal `.pystac`.

**Comment by alexgleith** [comment]
Yeah, no worries. Sorry it's not tested, it's a pretty complex environment for me still.

**Comment by gadomski** [comment]
Closing as OBE, https://github.com/stac-utils/stac-rs/pull/468

**Pull request closed without merging**